packages/ketcher-react/src/constants.ts

```diff
@@ -14,7 +14,8 @@
  * limitations under the License.
  ***************************************************************************/
 
-export const KETCHER_INIT_EVENT_NAME = 'ketcher-init';
+export const ketcherInitEventName = (ketcherId: string) =>
+  `ketcher-init-${ketcherId}`;
```

Comment:
Could you please make it backward-compatible by using old 'ketcher-init' id by default?